addons/binding/org.openhab.binding.keba/ESH-INF/thing/kecontact.xml

```diff
@@ -135,5 +136,11 @@
 		<description>Power consumption in Wh. Total energy consumption is added up after each completed charging session</description>
 		<state pattern="%d Wh" readOnly="true"></state>
 	</channel-type>
+    <channel-type id="display" advanced="true">
```

Comment:
If you use Eclipse with the ESH profile you should be able to format the XML (using build in functionallity) and it should format everything as the standard. All the lines of this channel type are indented with spaces instead of tabs.